pkg/application/application.go

```diff
@@ -107,64 +130,31 @@ func GenerateManifests(k *kusttypes.Kustomization) ([]byte, error) {
 
 /* CreateOptions impl */
 // Parse tries to parse `CreateOptions` into an `Application`.
-func (o *CreateOptions) Parse() (Application, error) {
-	return parseApplication(o)
+func (o *CreateOptions) Parse(ctx context.Context, co *git.CloneOptions, projectName string) (Application, error) {
```

Comment:
why do we need `context.Context`?
Can we send `repofs` instead of clone options?

Reply:
yeah, the ctx is not required anymore.
The cloneOptions are currently needed only for kustApp - to copy over the url and revision. I can just send these two values separately. Or maybe add them to the CreateOptions (along with the projectName)

Reply:
np, both sounds good to me